packages/core/src/global/motion.scss

```diff
@@ -0,0 +1,18 @@
+// Motion Duration Variables
+$motion-duration-instant: 0ms;
+$motion-duration-fast-01: 100ms;
+$motion-duration-fast-02: 150ms;
+$motion-duration-moderate-01: 200ms;
+$motion-duration-moderate-02: 300ms;
+$motion-duration-slow-01: 400ms;
+$motion-duration-slow-02: 500ms;
+
+// Motion Easing Variables
+$motion-easing-scania: cubic-bezier(0.4, 0, 0, 1);
+$motion-easing-enter: cubic-bezier(0.1, 0.9, 0.2, 1);
+$motion-easing-exit: cubic-bezier(0.7, 0, 1, 0.5);
+$motion-easing-easy: cubic-bezier(0.33, 0, 0.67, 1);
+$motion-easing-linear: cubic-bezier(0, 0, 1, 1);
+
+// Combined Motion Variables for Specific Animations
+$popover-core-transition: $motion-duration-moderate-01 $motion-easing-easy;
```

Comment:
As we talked, migrate the rule to the component-specific level as it is component-specific.

Reply:
Yes, I now populate the duration and easin directly in the component scss file 👍